urdf/ieg55.urdf.xacro

```diff
@@ -23,7 +23,7 @@
             </visual>
         </link>
 
-        <joint name="${prefix}jaw_position" type="prismatic">
+        <!-- <joint name="${prefix}jaw_position" type="prismatic">
```

Comment:
why?

Reply:
I guess since pitasc cannot/could not handle prismatic joints in the urdf description.

Reply:
it can? see `pitasc_common/examples/customized_robots/linear_axis.xml`

Reply:
added additional urdf with joints and removed the uncommented lines from the original file